returnn/datasets/util/vocabulary.py

```diff
@@ -162,12 +163,43 @@ def get_seq_indices(self, seq):
 
   def get_seq_labels(self, seq):
     """
-    :param list[int] seq:
+    :param list[int]|int|numpy.ndarray seq: 0D or 1D
     :rtype: str
     """
+    if isinstance(seq, int):
+      seq = [seq]
+    elif isinstance(seq, numpy.ndarray) and seq.ndim == 0:
+      seq = numpy.expand_dims(seq, axis=0)
     return " ".join(map(self.labels.__getitem__, seq))
 
 
+def get_seq_labels_from_labels(seq, labels):
+  """
+  Warning: Deprecated, use the corresponding vocabulary to serialize the data
```

Comment:
I would remove this warning. When it is used (directly or indirectly) and there is no real alternative (such as for `Dataset.labels`, `Dataset.serialize_data`), then it doesn't make sense to declare it as deprecated.

You could instead write sth like:
```suggestion
  In case you have a :class:`Vocabulary`, just use :func:`Vocabulary.get_seq_labels`.
```